Makefile

```diff
@@ -172,3 +174,16 @@ release-localkube: out/localkube checksum
 update-releases: 
 	gsutil cp deploy/minikube/k8s_releases.json gs://minikube/k8s_releases.json
 
+out/localkube-image: out/localkube
+	rm -rf out/localkube-image_tmp
```

Comment:
You might be able to simplify this a bit. What if you try:

* Copying deploy/ into out/
* docker build -t $TAG -f $PATHTODOCKERFILE <out>

Then you'll have access to the deploy/addons code from inside the dockerfile and you won't have to make a tmpdir.